Makefile

```diff
@@ -1,32 +1,38 @@
 # Setting variables
-
+SHELL := /bin/bash
 PIP_INSTALL := python3 -m pip install --no-cache-dir
 PIP_UNINSTALL := python3 -m pip uninstall
 
 ## Colorful output
-_INFO := $(shell echo -e '\e[1;36m')
-_WARNING := $(shell echo -e '\e[1;33m')
-_END := $(shell echo -e '\e[0m')
+_INFO := $(shell echo -e '\033[1;36m')
```

Comment:
have you done regression test on Linux about this colorful output?

Reply:
Yes, Linux also support it.
It is because in OS 'Esc' is only allowed to be encoded to '\033'. However in Linux both '\e' and '\033' are allowed.